### PR TITLE
removed default

### PR DESCRIPTION
**Pull request by ManiacMoustache**
## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] I have read the **CONTRIBUTING** document.
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

The previous version didn't allow us to choose autonomous modes due to overriding choices and all that
This removes the code that overrode the choice so that we can choose - requires the drivers to make sure they have the right autonomous mode though

### CODE REVIEW

**Comment by ChrisHirsch** [comment]
Can we fill out the entire PR so everyone knows what's happening?

File: robot.py [hunk 1]
```diff
@@ -110,8 +108,6 @@ def createObjects(self):
 
         self.driverStation = DriverStation.getInstance()
 
-        # Make sure default autonomous is big brain
```
Comment (ChrisHirsch) [inline robot.py:113]: How was this tested?

Comment (ManiacMoustache) [reply robot.py:113]: In sim, I didn't think to test it further than looking at the network tables
I should've checked states during autonomous

**Comment by ManiacMoustache** [comment]
Doesn't crash sim

**Review by ChrisHirsch — APPROVED** [review]
Reverted change and tested in sim 
+1 👍 🥇

**Review by Leaf-life — APPROVED** [review]
good

**Pull request merged**